src/test/shared/vscode/window.ts

```diff
@@ -66,6 +71,9 @@ export function createTestWindow(): Window & TestWindow {
             if (prop === 'showErrorMessage') {
                 return TestMessage.create(SeverityLevel.Error, fireOnDidShowMessage)
             }
+            if (prop === 'withProgress') {
+                return TestMessage.create(SeverityLevel.Progress, fireOnDidShowMessage)
```

Comment:
oh i think maybe this needs to be adjusted, that could avoid the hack mentioned above?

Reply:
this causes a hang..

```
2023-02-20T20:11:01.7995060Z       [32m  ✔[0m[90m uses `Timeout` objects[0m
2023-02-20T20:11:01.8998040Z rejected promise not handled within 1 second: Error: No return value has been set. Use `getOpenExternalStub().resolves` to set one.
2023-02-20T20:11:01.9100190Z stack trace: Error: No return value has been set. Use `getOpenExternalStub().resolves` to set one.
2023-02-20T20:11:01.9201490Z     at Context.<anonymous> (/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/src/test/globalSetup.test.ts:69:9)
2023-02-20T20:11:01.9300950Z     at Generator.next (<anonymous>)
2023-02-20T20:11:01.9401940Z     at /Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/dist/src/test/globalSetup.test.js:12:71
2023-02-20T20:11:01.9502230Z     at new Promise (<anonymous>)
2023-02-20T20:11:01.9603330Z     at __awaiter (/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/dist/src/test/globalSetup.test.js:8:12)
2023-02-20T20:11:01.9704000Z     at Context.<anonymous> (/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/dist/src/test/globalSetup.test.js:68:12)
2023-02-20T20:11:01.9805500Z     at callFn (/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/node_modules/mocha/lib/runnable.js:366:21)
2023-02-20T20:11:01.9906210Z     at Hook.Runnable.run (/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/node_modules/mocha/lib/runnable.js:354:5)
2023-02-20T20:11:02.0006880Z     at next (/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/node_modules/mocha/lib/runner.js:498:10)
2023-02-20T20:11:02.0107340Z     at Immediate.<anonymous> (/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/node_modules/mocha/lib/runner.js:559:5)
2023-02-20T20:11:02.0208070Z     at processImmediate (node:internal/timers:466:21)
2023-02-20T20:11:02.0308250Z     at process.topLevelDomainCallback (node:domain:152:15)
2023-02-20T20:11:02.0376510Z     at process.callbackTrampoline (node:internal/async_hooks:128:24)
2023-02-20T20:11:02.0433260Z No return value has been set. Use `getOpenExternalStub().resolves` to set one.: Error: No return value has been set. Use `getOpenExternalStub().resolves` to set one.
2023-02-20T20:11:02.0497170Z     at Context.<anonymous> (/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/src/test/globalSetup.test.ts:69:9)
2023-02-20T20:11:02.0598810Z     at Generator.next (<anonymous>)
2023-02-20T20:11:02.0670130Z     at /Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/dist/src/test/globalSetup.test.js:12:71
2023-02-20T20:11:02.0735410Z     at new Promise (<anonymous>)
2023-02-20T20:11:02.0837110Z     at __awaiter (/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/dist/src/test/globalSetup.test.js:8:12)
2023-02-20T20:11:02.0937790Z     at Context.<anonymous> (/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/dist/src/test/globalSetup.test.js:68:12)
2023-02-20T20:11:02.1038220Z     at callFn (/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/node_modules/mocha/lib/runnable.js:366:21)
2023-02-20T20:11:02.1140350Z     at Hook.Runnable.run (/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/node_modules/mocha/lib/runnable.js:354:5)
2023-02-20T20:11:02.1242340Z     at next (/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/node_modules/mocha/lib/runner.js:498:10)
2023-02-20T20:11:02.1344400Z     at Immediate.<anonymous> (/Users/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/node_modules/mocha/lib/runner.js:559:5)
2023-02-20T20:11:02.1445930Z     at processImmediate (node:internal/timers:466:21)
2023-02-20T20:11:02.1547390Z     at process.topLevelDomainCallback (node:domain:152:15)
2023-02-20T20:11:02.1648920Z     at process.callbackTrampoline (node:internal/async_hooks:128:24)
2023-02-20T20:11:02.1751540Z       [32m  ✔[0m[90m still runs if the timer completed (not rejected) after starting[0m[31m (92ms)[0m
2023-02-20T20:11:02.1853380Z       [32m  ✔[0m[90m rejects if using a completed timer[0m[33m (39ms)[0m
2023-02-20T20:11:02.1955000Z [0m    stop()[0m
2023-02-20T20:11:02.2046610Z     [32m  ✔[0m[90m detects running processes and successfully stops a running process - Unix[0m[33m (39ms)[0m
2023-02-20T20:11:02.2152570Z     [32m  ✔[0m[90m can stop() previously stopped processes - Unix[0m
2023-02-20T20:11:02.2237450Z     [32m  ✔[0m[90m can send input - Unix[0m
2023-02-20T20:11:02.2339050Z [0m  cliUtils[0m
2023-02-20T20:11:02.2440680Z [0m    installCli[0m
2023-02-20T20:54:23.0129460Z ##[error]The operation was canceled.
2023-02-20T20:54:23.0310590Z Post job cleanup.
```